src/communication/azext_communication/generated/_help.py

```diff
@@ -12,116 +12,125 @@
 from knack.help_files import helps
 
 
-helps['communication'] = """
+helps['communication communicationservice'] = """
     type: group
-    short-summary: Manage service with communication
+    short-summary: Manage communicationservice with communication
 """
 
-helps['communication list'] = """
+helps['communication communicationservice list'] = """
```

Comment:
I think communicationservice has prefix communication which is duplicated with parent command group name. Why not rename it to service as a short?

Reply:
I don't know what the guidelines for the naming is. I didn't make any customizations since the initial version, so I don't know why this generated differently. What should this look like? How can I get it to generate with the correct command names?

Reply:
Could you please share your swagger link and codegen configuration?